containers/Java/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/java/.devcontainer/base.Dockerfile
+
+# [Choice] Java version: 11, 16
```

Comment:
how did you know these two were the available versions?

Reply:
These values come from the 'built-in' container from VSCode repository